src/Content.stories.js

```diff
@@ -0,0 +1,11 @@
+import Content from './Content';
+
+export default {
+  title: 'Component/Content',
+  component: Content,
+};
+
+const Template = args => <Content {...args} />;
+
+export const Default = Template.bind({});
+Default.args = { children: 'Here will go content of the main area' };
```

Comment:
Does this work? Beacuse `<Content>` doesn't have any props

Reply:
It actually does :) Did the same for my button